notebooks/304-pytorch-post-training-quantization/304-pytorch-post-training-quantization.ipynb

```diff
@@ -0,0 +1,709 @@
+   "execution_count": null,
+   "id": "70ae8672",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if not fp32_ir_path.exists():\n",
+    "    !mo --input_model $fp32_onnx_path --input_shape $input_shape --mean_values \"[123.675, 116.28 , 103.53]\" --scale_values \"[58.395, 57.12 , 57.375]\" --data_type FP16 --output_dir $OUTPUT_DIR"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ab8be7f3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if not int8_ir_path.exists():\n",
+    "    !mo --input_model $int8_onnx_path --input_shape $input_shape --mean_values \"[123.675, 116.28 , 103.53]\" --scale_values \"[58.395, 57.12 , 57.375]\" --data_type FP16 --output_dir $OUTPUT_DIR"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cd1943fd",
+   "metadata": {},
+   "source": [
+    "## IV. Compare perfomance of INT8 model and FP32 model in OpenVINO"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ec2665fb",
+   "metadata": {},
+   "source": [
+    "Finally, we will measure the inference performance of the FP32 and INT8 models. To do this, we use [Benchmark Tool](https://docs.openvinotoolkit.org/latest/openvino_inference_engine_tools_benchmark_tool_README.html) - OpenVINO's inference performance measurement tool. By default, Benchmark Tool runs inference for 60 seconds in asynchronous mode on CPU. It returns inference speed as latency (milliseconds per image) and throughput (frames per second) values."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "83c0eba6",
+   "metadata": {},
+   "source": [
+    "> **NOTE**: In this notebook we run benchmark_app for 15 seconds to give a quick indication of performance. For more accurate performance, we recommended running benchmark_app in a terminal/command prompt after closing other applications. Run benchmark_app -m model.xml -d CPU to benchmark async inference on CPU for one minute. Change CPU to GPU to benchmark on GPU. Run benchmark_app --help to see an overview of all command line options.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6e5aada3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def parse_benchmark_output(benchmark_output):\n",
+    "    parsed_output = [line for line in benchmark_output if not (line.startswith(r\"[\") or line.startswith(\"  \") or line == \"\")]\n",
+    "    print(*parsed_output, sep='\\n')\n",
+    "\n",
+    "\n",
+    "print('Benchmark FP32 model (IR)')\n",
+    "benchmark_output = ! benchmark_app -m $fp32_ir_path -d CPU -api async -t 15\n",
+    "parse_benchmark_output(benchmark_output)\n",
+    "\n",
+    "print('Benchmark INT8 model (IR)')\n",
+    "benchmark_output = ! benchmark_app -m $int8_ir_path -d CPU -api async -t 15\n",
+    "parse_benchmark_output(benchmark_output)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3d29bc33",
+   "metadata": {},
+   "source": [
+    "Show CPU Information for reference"
```

Comment:
Colon missing at the end of this sentence

Reply:
Added